app/src/main/java/com/example/habitsmasher/ui/dashboard/HabitViewFragment.java

```diff
@@ -1,6 +1,10 @@
 package com.example.habitsmasher.ui.dashboard;
 
+import static android.content.ContentValues.TAG;
+
+import android.nfc.Tag;
 import android.os.Bundle;
+import android.util.Log;
```

Comment:
Are you actually using these imports or can they be removed?

Reply:
Removed 👍